src/chapters/loops/for-loops.rst

```diff
@@ -0,0 +1,90 @@
+**for** Loops
+=============
+
+.. index::
+   single: loop; for
+   single: iteration; definite
+
+.. index:: ! for loop
+
+The ``for`` loop is the first JavaScript tool for iteration that we will explore. A **for loop** allows us to carry out **definite iteration**. Definite iteration is the process of repeating a specific task with a specific data set. In other words, when a ``for`` loop begins it can usually be determined exactly how many times it will execute: once for each item in the data set.
```

Comment:
Clunky grammar.  Streamline to remove the need for “In other words”.

Reply:
I streamlined this paragraph, but left (most of) the last sentence. As opposed to Python (and some others) not _all_ `for` loops in JS are definite.